includes/class-wcmp-export.php

```diff
@@ -94,30 +94,12 @@ public function export() {
 		$order_ids = !empty($order_ids) ? $this->sanitize_posted_array($order_ids) : array();
 
 		switch($request) {
-			case 'add_shipments':
-				// filter out non-myparcel destinations
-				$order_ids = $this->filter_myparcel_destination_orders( $order_ids );
+            case 'get_labels':
+                $this->export_postnl_label($order_ids);
 
-				if ( empty($order_ids) ) {
-					$this->errors[] = __( 'You have not selected any orders!', 'woocommerce-myparcel' );
-					break;
-				}
-
-				// if we're going to print directly, we need to process the orders first, regardless of the settings
-				$process = (isset($print) && $print == 'yes') ? true : false;
-				$return = $this->add_shipments( $order_ids );
-				break;
-			case 'add_return':
-				if ( empty($myparcel_options) ) {
-					$this->errors[] = __( 'You have not selected any orders!', 'woocommerce-myparcel' );
-					break;
-				}
-				$return = $this->add_return( $myparcel_options );
-				break;
-			case 'get_labels':
 				$offset = !empty($offset) && is_numeric($offset) ? $offset % 4 : 0;
 				if ( empty($order_ids) && empty($shipment_ids)) {
```

Comment:
Deze check moet je plaatsen voor het aanroepen van export_postnl_label()

Reply:
Deze check staat dubbel. Het staat ook in export_postnl_label(). Daar moet je het weghalen. Zie ander comment.